pontoon/base/migrations/0045_auto_20151112_0857.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
I've tried to make a habit of renaming auto-named migrations with a more descriptive name, it helps make things clearer.